### DISCUSSION TITLE
[Announcement] Warming-up Homework

### DISCUSSION THREAD

**Discussion by Junseon00** [prosyslab-classroom/cs348-information-security#439] [Announcements]
Goal: setting up and getting familiarized with OCaml and Git

- Implement simple programs in OCaml (e.g., hello-world)
- Push to your Github repository
- Will not be graded

Invitation URLs

1. hello-world: https://classroom.github.com/a/WupWHckw
2. ocaml-programming: https://classroom.github.com/a/1r_BQoss

**Comment by nv-quan** [comment]
Does anyone have problems with running Travis CI? I applied the Student Pack but it still refuse to run. "Builds have been temporarily disabled for private repositories due to a negative credit balance. "

**Comment by miraliahmadli** [comment]
> Does anyone have problems with running Travis CI? I applied the Student Pack but it still refuse to run. "Builds have been temporarily disabled for private repositories due to a negative credit balance. "

I have the same issue as well. In plan page, it shows I still have 10K credits given by Github.

**Comment by KihongHeo** [comment]
Hello, guys.
Sorry for confusion. Please forget about travis for now. It will not play any critical role in our homework. It worked well previously but I noticed the issue right before this semester.

I have asked Travis to fix the issue several time but they do not respond.
